src/js/chartAuthoring.js

```diff
@@ -237,6 +360,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             $(this).find(dataEntryLabelSelector).val(currentData.label).trigger("change");
             $(this).find(dataEntryValueSelector).val(currentData.value).trigger("change");
         });
+
+        that.events.onUpdateDataEntryPanel.fire();
+    };
+
+    floe.chartAuthoring.addSonificationControlListeners = function(that) {
```

Comment:
@cindyli - works for me. I should remember that "listener" has a very specific meaning in Infusion and avoid using it in the more general sense in favour of "handler".